src/components/Buttons/ButtonBlue.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+const ButtonBlue = ({title}) => {
+  return <StyledButton type="button">{title}</StyledButton>;
+};
+
+export default ButtonBlue;
+
+const StyledButton = styled.button`
```

Comment:
Most (if not all) of these buttons can be refactored into one button components, with multiple variants for styling. Fx something like:

`const ButtonGeneral = ({ title, buttonVariation }) => {
  // Map variations to styled components
  const buttonStyles = {
    variation1: ButtonBlue,
    variation2: ButtonPurple,
    // Add more variations as needed + DefaultStyledButton
  };

  // Select the appropriate styled component
  const StyledButton = buttonStyles[buttonVariation] || DefaultStyledButton;

  return <StyledButton type="button">{title}</StyledButton>;
};

export default ButtonGeneral;

// Define styled components for each variation :) 
`